packages/d/dom-to-image-more.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "dom-to-image-more",
+  "filename": "dom-to-image-more.min.js",
+  "description": "Generates an image from a DOM node using HTML5 canvas and SVG",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/1904labs/dom-to-image-more.git"
+  },
+  "keywords": [
+    "dom",
+    "image",
+    "raster",
+    "render",
+    "html",
+    "canvas",
+    "svg"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "dom-to-image-more",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js?(.map)"
```

Please avoid using a globstar as it can lead to including files you did not intend to include on the CDN -- looking at the package, `*.js?(.map)` would suffice here